lib/plugins/aws/package/compile/functions/index.test.js

```diff
@@ -25,6 +25,9 @@ describe('AwsCompileFunctions', () => {
   let awsCompileFunctions;
   const functionName = 'test';
   const compiledFunctionName = 'TestLambdaFunction';
+  let exampleCompiledLayerResource;
```

Comment:
Let's write new tests only with help of `runServerless` util. See https://github.com/serverless/serverless/tree/master/tests#unit-tests

I think in tests we should just confirm on `CodeHash256`, and confirm that it changes when expected an stay same when expected.

We may also prepare a new fixture that has some minimal layer